.github/workflows/diff_report.yml

```diff
@@ -64,17 +70,27 @@ jobs:
   make_report:
     runs-on: ubuntu-latest
     needs: parse_body
-    if: needs.parse_body.outputs.project_link != '' && needs.parse_body.outputs.config_link != ''
+    if: needs.parse_body.outputs.projects_link != '' &&
+        (needs.parse_body.outputs.config_link != '' ||
```

Comment:
please move `&&` and `||` to the begining

Reply:
done